bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/TupleValueImpl.java

```diff
@@ -163,6 +163,11 @@ public Object getRefValue(long index) {
         return get(index);
     }
 
+    @Override
+    public Object fillAndGetRefValue(long index) {
+        return get(index);
```

Comment:
Shouldn't we implement the same for tuples too?

The following fails with an index out of range?

```ballerina
public function main() {
    [boolean, int[]...] x = [true];

    x[1][1] = 1;
}
```

The following also
```ballerina
public function main() {
    [boolean, [float, int]...] x = [true];

    x[1][1] = 1;
}
```

Reply:
This was fixed by adding a fillRead() method for tuple as well.